TJDropbox.podspec

```diff
@@ -0,0 +1,21 @@
+Pod::Spec.new do |s|
+  s.name         = "TJDropbox"
+  s.version      = "0.4"
+  s.summary      = "A Dropbox v2 client library written in Objective-C"
+  s.description  = "Dropbox provides a v2 SDK for iOS, but it's only compatible with Swift."
+
+  s.homepage     = "https://github.com/timonus/TJDropbox"
+
+  s.license      = "BSD 3-clause \"New\" or \"Revised\" License"
+
+  s.author       = "Tim Johnsen"
+
+  s.source       = { :git => "https://github.com/timonus/TJDropbox.git", :tag => "0.4" }
+
+  s.source_files = "TJDropbox/*.{h,m}"
+
+  s.ios.deployment_target = "8.0"
+  # s.osx.deployment_target = "10.9"
```

Comment:
TJDropbox actually already supports macOS 😄 but TJDropboxAuthenticationViewController.h/m needs to be excluded. I'm not familiar with CocoaPods, but could you update to add that?